Ghidra/Features/PDB/src/pdb/cpp/find.cpp

```diff
@@ -15,97 +15,119 @@
  * limitations under the License.
  */
 #include "find.h"
+#include <atlcomcli.h>
+#include <comutil.h>
 
-BSTR findMangledName(IDiaSymbol * pFunction) {
-	DWORD rva = getRVA(pFunction);
-	IDiaSymbol * pSymbol = NULL;
-	HRESULT hr = pSession->findSymbolByRVA(rva, SymTagPublicSymbol, &pSymbol);
-	if (hr == S_OK) {
-		DWORD tag = getTag(pSymbol);
+std::wstring findMangledName(PDBApiContext& ctx, IDiaSymbol& pFunction) {
+	const DWORD rva = getRVA(pFunction);
+	CComPtr<IDiaSymbol> pSymbol;
+	const HRESULT hr = ctx.Session().findSymbolByRVA(rva, SymTagPublicSymbol, &pSymbol);
```

Comment:
Combine lines 24 and 25, eliminating HRESULT hr: if (SUCCEEDED(ctx.Session()...))